configuration.yml

```diff
@@ -19,7 +19,7 @@ alluxio_port: "19998"
 pyspark_cassandra_version: "0.3.5"
 analysers_path: "/app/analysers"
 transformers_path: "/app/data-transformers"
-configurations_path: "/app/configuration"
+configurations_path: "/app/data-transformers/suts/wfms"
```

Comment:
@VincenzoFerme And where do we say that we are dealing with a wfms? Cause right now the only thing passed dynamically is SUTName and SUTVersion, and they don't specifiy it